inst/pages/Introductory-workflow.qmd

```diff
@@ -0,0 +1,353 @@
+ggplotly(p)
+```
+
+However, the axes are not very informative and the amount of captured variance by the algorithm is nowhere to be found. We can tweak the plot to show some more information as such:
+
+```{r}
+#| label: addVariancePCoA
+# Calculate explained variance
+e <- attr(reducedDim(tse, "MDS_bray"), "eig")
+rel_eig <- e / sum(e[e > 0])
+
+# Add explained variance for each axis on the plot
+p <- p + labs(x = paste("PCoA 1 (", round(100 * rel_eig[[1]], 1), "%", ")", sep = ""),
+              y = paste("PCoA 2 (", round(100 * rel_eig[[2]], 1), "%", ")", sep = ""))
+
+# Reonvert to an interactive plot with ggplotly
+ggplotly(p)
+```
+
+There we have it! Each axis shows the amount of variance or in our case dissimilarity retained by each principal coordinate. You can additionally add more options to color by a certain characteristic for example. More on this can be found in @sec-comparing-communities-by-beta-diversity-analysis.
+
+<hr>
+
+# Datavisualization using heatmaps
+
+Heatmaps are one of the most versatile ways to visualise your data. In this section we'll cover how to create a basic heatmap to visualise the most prevalent features using the [**sechm library**](https://bioconductor.org/packages/release/bioc/html/sechm.html). For a more detailed heatmap, please revert to [**this section**](https://microbiome.github.io/OMA/docs/devel/pages/23_multi-assay_analyses.html) of the OMA book.
+
+Let's first load the library:
+
+```{r}
+#| label: load-sechm
+#| output: false
+library(sechm)
+```
+
+Then, let's subset the TreeSE to the most prevalent taxa:
+
+```{r}
+#| label: most-prevalent
+tse.subset <- subsetByPrevalent(tse,prevalence=0.5)[1:5,]
+```
+
+When subsetting with this function, the resulting object no longer contains the correct relative abundances because these abundances were originally calculated based on the full dataset, not the subsetted one. Therefore, we need to recalculate the values to reflect the subset accurately. For the sake of readability, we'll also subset to the first five taxa after the initial subset.
+
+```{r}
+#| label: recalculate-relabundance
+tse.subset <- transformAssay(tse.subset, assay.type = "counts", 
+                             method = "relabundance")
+
+```
+
+Now that we have prepared the data, we can use the previously loaded sechm library to plot the heatmap:
+
+```{r}
+#| label: heatmap
+# Sets the colors
+setSechmOption("hmcols", value=c("#F0F0FF","#007562"))
+
+# Plots the actual heatmap.
+sechm(tse.subset, features = rownames(rowData(tse.subset)),
+      assayName="relabundance",show_colnames=TRUE)
+```
+
+In the heatmap above, it is apparent that Parabacteroides are relatively common in some samples while Akkermansia is detected very seldom.
+
+<h2>Acknowledgements</h2>
+
+This notebook has been carefully crafted by:
+
+![](images/profile-pics/axel.webp){.profilepic} **Axel Dagnaud**
+
+![](images/profile-pics/noah.png){.profilepic} **Noah de Gunst**
+
+-   **Leo Lahti**
```

Comment:
All the authors are listed in https://microbiome.github.io/OMA/docs/devel/pages/90_acknowledgments.html so add your names there instead if they are not there already